modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -650,6 +692,35 @@ impl WalletClientModule {
         ))
     }
 
+    pub async fn peg_in(&self, req: PegInRequest) -> anyhow::Result<PegInResponse> {
+        let (operation_id, address, _) = self.safe_allocate_deposit_address(req.extra_meta).await?;
+        self.pegin_monitor_wakeup_sender
+            .send(())
+            .context("Failed to wake up peg-in monitor")?;
```

Comment:
Somehow CI didn't catch this banned method invocation (`send` instead of `send_replace`)